main.py

```diff
@@ -141,25 +142,34 @@ def get_inputfiles(args):
 
     return inputfiles
 
+def ats(imagefile, colony, args, config):
+    temperature, end_temperature = auto_temp_schedule(imagefile, colony, args, config)
+    print("auto temperature schedule started")
+    setattr(args, 'start_temp', temperature)
+    setattr(args, 'end_temp', end_temperature)
+    print("auto temperature schedule finished")
```

Comment:
I just noticed that `print("auto temperature schedule started")` comes after the call to `auto_temp_schedule`. Also, since we are already planning on inlining this function, we can change the `setattr` calls.
```suggestion
    print("auto temperature schedule started")
    temperature, end_temperature = auto_temp_schedule(imagefile, colony, args, config)
    args.tart_temp = temperature
    args.end_temp = end_temperature
    print("auto temperature schedule finished")
```